heron/config/src/yaml/conf/yarn/uploader.yaml

```diff
@@ -1,5 +1,5 @@
 # uploader class for transferring the topology jar/tar files to storage
-heron.class.uploader:                            com.twitter.heron.uploader.NullUploader
+heron.class.uploader:                            com.twitter.heron.uploader.localfs.LocalFileSystemUploader
```

Comment:
which means it will copy to another directory - which incurs overhead and also a configuration of directory where the file needs to be moved as well. Probably, it might be a good idea to have NullUploader.